caf-audit-binding-webservice/src/main/java/com/hpe/caf/auditing/webserviceclient/WebServiceClientAuditEventBuilder.java

```diff
@@ -193,7 +194,7 @@ public void send() throws IOException, WebServiceClientException {
         if (auditEventJson == null || auditEventJson.isEmpty()) {
             String errorMessage = "No Audit Event JSON to send to the Audit WebService";
             LOG.error(errorMessage);
-            throw new WebServiceClientException(errorMessage);
+            throw new AuditFailureException(errorMessage);
```

Comment:
Is this not just library misuse?  If so then it should be some type of unchecked exception thrown here.